.env.example

```diff
@@ -5,4 +5,4 @@ SMTP_HOST="smtp.example.com"
 OTP_SECRET="lorememsum"
 JWT_SECRET="yoursecert"
 PASS_RESET_LINK="http://localhost:5173/reset-password"
-ALOOWED_ORIGIN="http://localhost:3000,http://yourdomain.com"
\ No newline at end of file
+ALLOWED_ORIGIN="http://localhost:3000,http://yourdomain.com"
```

Comment:
The spelling here is different from what you used in `constant.rs`. 
Change it to `ALLOWED_ORIGINS`